html2text/utils.py

```diff
@@ -254,6 +254,16 @@ def reformat_table(lines, right_margin):
     max_width = [len(x.rstrip()) + right_margin for x in lines[0].split('|')]
     for line in lines:
         cols = [x.rstrip() for x in line.split('|')]
+
```

Comment:
Would be much cleaner to keep the length of `max_width` and `cols` in a variable so you don't have to write `len(cols)` and `len(max_width)` in multiple places.

like:
```python
cols_size =len(cols) 
max_width_sizet =len(max_width)
```

Something like that would be much cleaner.

Reply:
No worries, how's that now?